sql/core/src/test/resources/sql-tests/results/postgreSQL/numeric.sql.out

```diff
@@ -4793,33 +4793,43 @@ Infinity
 -- !query
 select * from range(cast(0.0 as decimal(38, 18)), cast(4.0 as decimal(38, 18)))
 -- !query schema
-struct<id:bigint>
+struct<>
 -- !query output
-0
-1
-2
-3
+org.apache.spark.sql.AnalysisException
+Table-valued function range with alternatives: 
+    range(start: long, end: long, step: long, numSlices: integer)
+    range(start: long, end: long, step: long)
+    range(start: long, end: long)
+    range(end: long)
+cannot be applied to (decimal(38,18), decimal(38,18)): Incompatible input data type. Expected: long; Found: decimal(38,18); line 1 pos 14
```

Comment:
cc @cloud-fan when the ansi mode enabled, we can't implicitly cast decimal(38, 18) to long.